### ISSUE TITLE
[Gate Proposal] Bundled null cell

### ISSUE THREAD

**Issue by joshop** [MrTJP/ProjectRed#1507]
It's slightly annoying when you must cross two bundled cables in an IC but have to split each into its components and cross them with null cells. A gate that crosses two bundled cables would be handy.

**Comment by daniel-pg** [comment]
I agree with you. In my opinion, there should also be a "Mixed Null Cell" that crosses a bundled cable and an insulated/red alloy wire.
